src/imgui/imguiWrapper/colors.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef CASH_OVERFLOW_COLORS_H
+#define CASH_OVERFLOW_COLORS_H
+
+#include <imgui.h>
+
+namespace cash_overflow::colors {
+
+class Color
+{
+public:
+  Color() : color{ 0.0f, 0.0f, 0.0f, 0.0f } {};
+  Color(float r, float g, float b, float o) : color{ r, g, b, o } {};
+  ImVec4 getColor() const
+  {
+    return color;
+  }
+
+private:
+  ImVec4 color{};
+};
+
+  const Color Black{ .0F, .0F, .0F, 1.0F };
```

Comment:
These constants in the header could break compilation if two compilation units include this header and then get linked. Use inline, static or constexpr. Or put it inside a static member function.

Reply:
I was wrong. We tested and this does work.